flow/environments/incite.py

```diff
@@ -21,7 +21,7 @@ class SummitEnvironment(DefaultLSFEnvironment):
         @directives(ngpu=3) # 3 GPUs
         @directives(np=3) # 3 CPU cores
         @directives(rs_tasks=3) # 3 tasks per resource set
-        @directives(extra_jsrun_args='-smpiargs="-gpu"') # extra jsrun arguments
+        @directives(extra_jsrun_args='--smpiargs="-gpu"') # extra jsrun arguments
```

Comment:
How is this related to the fix?

Reply:
> How is this related to the fix?

It is only related in that it pertains to the Summit template. The double dash is necessary for jsrun, the previous example contained a mistake.